manifests/01-registry-credentials-request-alibaba.yaml

```
@@ -0,0 +1,45 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: openshift-image-registry-alibaba
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  secretRef:
+    name: installer-cloud-credentials
+    namespace: openshift-image-registry
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AlibabaCloudProviderSpec
+    statementEntries:
+    - effect: Allow
+      action:
+      - oss:PutBucket
+      - oss:CreateBucket
+      - oss:DeleteBucket
+      - oss:PutBucketTags
+      - oss:GetBucketTags
+      - oss:PutBucketAcl
+      - oss:GetBucketAcl
+      - oss:PutBucketEncryption
+      - oss:GetBucketEncryption
+      - oss:PutBucketLifecycle
+      - oss:GetBucketLifecycle
+      - oss:GetBucketLocation
+      - oss:GetBucketInfo
+      - oss:ListBucket
+      - oss:GetObject
+      - oss:PutObject
+      - oss:DeleteObject
+      - oss:ListMultipartUploads
+      - oss:AbortMultipartUpload
+      - oss:ListParts
```

@kwoodson @menglingwei according with [official doc](https://www.alibabacloud.com/help/doc-detail/42777.htm), those permissions are missing:
```suggestion
      - oss:ListParts
      - oss:GetBucketReferer
      - oss:SetBucketReferer
```

---

@menglingwei I believe we are missing a few permissions here. Can you please add the necessary items for `oss:List*`? QE is running into a few issues when running this code without the permissions.

Thanks!